### ISSUE TITLE
Add followed games to the favourites tab

### ISSUE THREAD

**Issue by elegantandrogyne** [vinszent/gnome-twitch#161]
The favourites tab lists only followed hosts/channels (as of 0.2.1) - add also followed games.

**Comment by vinszent** [comment]
Yep this is planned, however I'm thinking of just placing followed channels at the top of the games page that are always there (except when searching). Kind of like pinned tabs in a browser.
